### PR TITLE
Add assigned CVE ID

### PR DESCRIPTION
**Pull request by ennru**
- https://akka.io/security/akka-http-cve-2023-05-15.html
- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-33251

### CODE REVIEW

**Review by patriknw — APPROVED** [review]
LGTM, can you update the release notes at the same time? https://akka.io/blog/news/2023/05/16/akka-23.5-released

File: blog/_posts/2023-05-16-akka-23.5-released.md [hunk 1]
```diff
@@ -69,6 +69,7 @@ We run automated security vulnerability and license policy scanning of dependenc
 
 The following CVE's are addressed in this build.
 
+* [CVE-2023-33251](https://akka.io/security/akka-http-cve-2023-05-15.html)
 * [CVE-2023-29471](https://akka.io/security/alpakka-kafka-cve-2023-29471.html)
 * [CVE-2023-31442](https://akka.io/security/akka-async-dns-2023-31442.html)
 * [Akka HTTP file upload](https://akka.io/security/akka-http-cve-2023-05-15.html)
```
Comment (patriknw) [inline blog/_posts/2023-05-16-akka-23.5-released.md:75]: it's already listed here, so just change the text `Akka HTTP file upload` to the CVE number

Comment (ennru) [reply blog/_posts/2023-05-16-akka-23.5-released.md]: Sorry for not paying enough attention.

**Review by patriknw — APPROVED** [review]
LGTM

**Pull request merged**